README.md

```diff
@@ -48,6 +48,7 @@ Operating an EVM Gateway is straightforward. It can either be deployed locally a
 --coinbase {EVM-account} \
 --coa-address {funded Flow account address} \
 --coa-key-file {file containing private keys for coa-address, KMS should be used on live networks} \
+--init-cadence-height {network Flow height from where to start indexing}
```

Comment:
Do we want to add the specific numbers that apply for testnet (and then mainnet when we have it)?

Reply:
That's a good point indeed. The minimum Cadence height will likely be the height in which the `EVM` contract was deployed on a given network (testnet/mainnet/previewnet etc).